api/envoy/data/extensions/filter/udp/dns_filter/v2alpha/inline_dns_table.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+package envoy.data.extensions.filter.udp.dns_filter.v2alpha;
+
+import "google/protobuf/duration.proto";
+
+import "udpa/annotations/status.proto";
+
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.data.extensions.filter.udp.dns_filter.v2alpha";
+option java_outer_classname = "InlineDnsTableProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).work_in_progress = true;
+
+// [#protodoc-title: DNS Filter Table Data]
+// :ref:`DNS Filter config overview <config_udp_listener_filters_dns_filter>`.
+
+// This message contains the configuration for the DNS Filter if populated
+// from the control plane
+message InlineDnsTable {
+  // This message contains a list of IP addresses returned for a query for a known name
+  message AddressList {
+    repeated string address = 1;
```

Comment:
I'm assuming you didn't use the Envoy address type here since it includes info you don't need, but I'm wondering if we should be a bit more structured here around A vs. AAAA records, etc.? Or do you want to assume the string will encode v4 vs. v6 per normal string rules? Either way can you clarify the docs?

Reply:
I tried `Address` but I wasn't a fan of the level of nesting in the config.  That may or may not be a "real" concern. 

I can certainly add more to the docs.

Reply:
Yeah I would just add more docs here.

Reply:
Will do.